packages/material-ui/src/TextareaAutosize/TextareaAutosize.js

```diff
@@ -60,8 +62,8 @@ const TextareaAutosize = React.forwardRef(function TextareaAutosize(props, ref)
     // The height of the outer content
     let outerHeight = innerHeight;
 
-    if (rows != null) {
-      outerHeight = Math.max(Number(rows) * singleRowHeight, outerHeight);
+    if (rowsMin != null) {
```

Comment:
I think this can be change to simply `rowsMin` since null is falsy or use the `!==` operator instead

Reply:
Sounds good. We don't really need to support 0.